packages/integration/pubspec.yaml

```diff
@@ -33,7 +33,7 @@ dependencies:
   media_kit_video: ^1.2.4 # For video rendering.
   media_kit_libs_video: ^1.0.4
   flutter_keyboard_visibility: ^5.4.1
-  extended_text_field: ^12.0.1
+  extended_text_field: ^14.0.0
```

Comment:
we should support the min version, and have it overridden in case we have to support it on a newer version

Reply:
When you tried it on your end did it break like expected?